repository/postgresql/user_test.go

```diff
@@ -0,0 +1,257 @@
+package repository_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/docker/go-connections/nat"
+	"github.com/jackc/pgx/v4/pgxpool"
+	"github.com/serhiihuberniuk/blog-api/models"
+	repository "github.com/serhiihuberniuk/blog-api/repository/postgresql"
+	"github.com/stretchr/testify/assert"
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+func createPostgresTestContainer(ctx context.Context) (func(), *repository.Repository, error) {
```

Comment:
hm, you can not return an error and accept testing.T, and if something not good -> just fail the test.

Reply:
close function usually return as lastes return value, before an error